.linthtmlrc.json

```diff
@@ -38,7 +38,7 @@
   ],
   "tag-name-lowercase": true,
   "tag-name-match": true,
-  "tag-self-close": "never",
+  "tag-self-close": false,
```

Comment:
Why did you changed this rule?

Reply:
😳😳😳 I didn't add changes in this file

Reply:
What i need to do? Change 'false' to 'never' or don't touch it?

Reply:
Change 'false' to 'never'  please